lib/lightning_web/live/job_live/job_builder.ex

```diff
@@ -137,12 +144,14 @@ defmodule LightningWeb.JobLive.JobBuilder do
                     module={LightningWeb.JobLive.AdaptorPicker}
                     on_change={fn adaptor -> send_adaptor(@job_id, adaptor) end}
                     form={f}
+                    can_edit_job={@can_edit_job}
```

Comment:
Rather pass through `disabled` instead of `can_edit_job`, I don't see why the component needs to know this - the pattern you have so far for everything else uses `disabled` no need to break that now.